scripts/lib-franklin.js

```diff
@@ -615,6 +615,13 @@ export function decorateButtons(element) {
  * Load LCP block and/or wait for LCP in default content.
  */
 export async function waitForLCP(lcpBlocks, skipBlocks = [], maxCandidates = 1) {
+  // Adding blocks containing hero variant in lcp blocks at runtime
+  document.querySelectorAll('[class*="hero-"]')
```

Comment:
I think the `lcpblocks` already has few `hero-` type blocks. Would the below code lead to duplicate entries? Also it would be a bit better if we had this logic moved to `loadEager` where we finalize the list of `lcpBlocks`.